meetings/2020-11-11.md

```diff
@@ -0,0 +1,58 @@
+# Node.js Technical Steering Committee (TSC) Meeting 2020-11-11
+
+## Links
+
+* **Recording**: https://youtu.be/z4n-K0m_cro
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/949
+
+## Present
+
+* Beth Griggs @BethGriggs (TSC)
+* Ruben Bridgewater @BridgeAR (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* Daniel Bevenius @danbev (TSC)
+* Gireesh Punathil @gireeshpunathil (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Rich Trott @Trott (TSC)
+
+## Agenda
+
+### Announcements
+
+* Reminder to TSC members to update the availability matrix.
+
+### CPC and Board Meeting Updates
+ 
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+* No updates this week 
+
+### nodejs/node
+
+* Backporting `util/types`, `path/posix` and `path/win32` to v15 proposal [#35788](https://github.com/nodejs/node/issues/35788)
+  * This looks like it can be close as the referenced PRs are landed.
+  * Michael to close
```

Comment:
This resolution was changed in <https://github.com/nodejs/node/issues/35788#issuecomment-725534423>.

Reply:
/cc @guybedord who I talked about this with

Reply:
I think you meant @guybedford 😄